src/main/java/com/boardgo/domain/meeting/entity/MeetingEntity.java

```diff
@@ -77,10 +77,14 @@ public class MeetingEntity extends BaseEntity {
 
     @Column(nullable = false)
     @Enumerated(EnumType.STRING)
-    MeetingState state;
+    private MeetingState state;
+
+    @Column
+    @ColumnDefault("0")
+    private Long shareCount;
```

Comment:
Bigint 크기까지는 안필요할 것 같아서 Integer 로 하는게 어떨가용?
```suggestion
    @ColumnDefault("0")
    private Integer shareCount;
```

Reply:
넵 좋습니다!